adafruit_lsm303.py

```diff
@@ -237,6 +252,20 @@ def mag_rate(self, value):
         reg_m = ((value & 0x07) << 2) & 0xFF
         self._write_u8(self._mag_device, _REG_MAG_CRA_REG_M, reg_m)
 
+    @property
+    def accel_rate(self):
+        """The accelerometer update rate."""
+        return self._accel_rate
+
+    @accel_rate.setter
+    def accel_rate(self, value):
+        assert value in (ACCELRATE_1, ACCELRATE_10, ACCELRATE_25, ACCELRATE_50, ACCELRATE_100,  ACCELRATE_200, ACCELRATE_400, ACCELRATE_1620, ACCELRATE_1344)
```

Comment:
Two things here:

1. This is where the Travis check is failing. The line is too long at 175 characters (100 characters is pylint's standard), and there are two spaces before `ACCELRATE_200`. If this shows as two lines in your editor, it may be a linefeed encoding issue.

2. Can you please add an exception message for the `assert`, so that users will know why it fails. Example on a Feather M0:
```
Adafruit CircuitPython 3.0.2 on 2018-09-14; Adafruit Feather M0 Express with samd21g18
>>> assert_tuple = (0,1,2,3)
>>> assert 15 in assert_tuple
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError:

>>> assert 15 in assert_tuple, "value must be one of the following: {}".format(assert_tuple)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: value must be one of the following: (0, 1, 2, 3)
>>>
```

Reply:
On second thought with number 2 above, it may be better to switch to an `if` condition, and raise a `ValueError`. An `AssertionError` is a bit esoteric from a user's perspective; especially for beginners.
```
valid_values = (ACCELRATE_1, ACCELRATE_10, ACCELRATE_25, ACCELRATE_50, ACCELRATE_100,
                ACCELRATE_200, ACCELRATE_400, ACCELRATE_1620, ACCELRATE_1344)
if value not in valid_values:
    raise ValueError("Provided rate invalid. Must be one of the following: {}".format(valid_values))
```
Thoughts on this perspective, @adafruit/circuitpythonlibrarians?

Reply:
Thank you a lot @sommersoft and the adafruit team for the replies! Fixed and repulling!

@sommersoft I like your idea a lot, I did think that it is very obfuscating to just raise an `AssertionError` however I kept the original style from the adafruit team like in the lines above:

`
assert value in (MAGGAIN_1_3, MAGGAIN_1_9, MAGGAIN_2_5, MAGGAIN_4_0, MAGGAIN_4_7, MAGGAIN_5_6, MAGGAIN_8_1)
`

but I adore your style of coding and I definitely will embrace this beautiful idea then!